feature-detects/mediastream.js

```diff
@@ -0,0 +1,13 @@
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', function(){
+        var isMediaRecorderSupported = false;
```

Comment:
> Follow Patrick's advice about with the typeof, I leave this here in case you wanted some insight about funcions ;)

At the start of the test you start with a function, functions have to return something (in this case `false` if the feature is not supported and `true` if it is supported). So instead of the creating a variable just return the value (the true case does not need too much tinkering, and the false case should go where the `console.log("no MediaRecorder");`)
If you need to [learn about functions I would recommend you the W3Schools tutorial](https://www.w3schools.com/js/js_functions.asp). Functions are super useful in the programming sphere so it is worth the time to learn how to use them 😊